core/psh/ping/ping.c

```diff
@@ -247,29 +249,31 @@ int psh_ping(int argc, char **argv)
 		return -EINVAL;
 	}
 
-	if ((fd = psh_ping_socketConfig()) <= 0)
+	if ((fd = ping_sockconf()) <= 0)
 		return -EIO;
 
-	psh_ping_reqInit(req, sizeof(req));
+	ping_reqinit(req, sizeof(req));
 
 	while (ping_common.cnt-- && !psh_common.sigint) {
 		clock_gettime(CLOCK_MONOTONIC, &ts);
 		sent = ts.tv_sec * 1000000 + ts.tv_nsec / 1000;
-		if (psh_ping_send(fd, req, sizeof(req)) < 0)	{
+		if (ping_send(fd, req, sizeof(req)) < 0) {
 			close(fd);
 			fprintf(stderr, "ping: Fail to send a packet!\n");
 			return -EINVAL;
```

Comment:
not related to this PR, but might be fixed here... don't return negative values from `main()` as it's not well-defined